app/views/static_pages/contributing/_hall_of_fame.erb

```diff
@@ -1,139 +1,31 @@
-      image: 'hall_of_fame_images/joshua.jpg',
-      github_url: 'http://github.com/gorchov',
-    },
-    {
-      name: 'Neil Gehani',
-      image: 'hall_of_fame_images/neil.jpg',
-      github_url: 'http://github.com/ngehani',
-    },
-    {
-      name: 'Ian White',
-      image: 'hall_of_fame_images/ian.jpg',
-      github_url: '',
-    },
-    {
-      name: 'Bill Walker',
-      image: 'hall_of_fame_images/bill.jpg',
-      github_url: 'http://github.com/mach1010',
-    },
-    {
-      name: 'Mazin Power',
-      image: 'https://avatars1.githubusercontent.com/u/7075351?v=3&s=400',
-      github_url: 'https://github.com/muzfuz',
-    },
-    {
-      name: 'Ada Chiu',
-      image: 'https://avatars0.githubusercontent.com/u/13413986?v=3&s=400',
-      github_url: 'https://github.com/adachiu',
-    },
-    {
-      name: 'Kevin Mulhern',
-      image: 'https://avatars1.githubusercontent.com/u/7963776?v=3&s=460',
-      github_url: 'https://github.com/KevinMulhern',
-    },
-    {
-      name: 'Arun Kumar',
-      image: 'https://avatars0.githubusercontent.com/u/7039523?v=3&s=400',
-      github_url: 'https://github.com/arku',
-    },
-    {
-      name: 'Cody Loyd',
-      image: 'https://avatars0.githubusercontent.com/u/18574792?v=3&s=400',
-      github_url: 'https://github.com/codyloyd',
-    },
-    {
-      name: 'Cornelius Soon',
-      image: 'https://avatars2.githubusercontent.com/u/16495574?v=3&s=400',
-      github_url: 'https://github.com/csrail',
-    },
-    {
-      name: 'Rhys',
-      image: 'https://avatars3.githubusercontent.com/u/19835799?v=3&s=400',
-      github_url: 'https://github.com/105ron',
-    },
-    {
-      name: 'Austin',
-      image: 'https://avatars2.githubusercontent.com/u/12649577?v=3&s=400',
-      github_url: 'https://github.com/CouchofTomato',
-    },
-    {
-      name: 'Leo',
-      image: 'https://avatars2.githubusercontent.com/u/13614862?v=3&s=400',
-      github_url: 'https://github.com/leosoaivan',
-    },
-    {
-      name: 'Chad Kreutzer',
-      image: 'https://avatars1.githubusercontent.com/u/13308917?v=3&s=400',
-      github_url: 'https://github.com/ChadKreutzer',
-    },
-    {
-      name: 'Roberto Pando',
-      image: 'https://avatars1.githubusercontent.com/u/20132181?v=3&s=400',
-      github_url: 'https://github.com/RobPando',
-    },
-    {
-      name: 'Ryan',
-      image: 'https://avatars3.githubusercontent.com/u/20901017?v=3&s=400',
-      github_url: 'https://github.com/ryanford-frontend',
-    }
-  ]
-%>
+<section class="pt-16 gradient">
+  <div class='page-container'>
+    <h3 class="text-gold text-center font-bold pb-7 text-2xl">Hall of Fame</h3>
+    <p class="text-center max-w-prose mx-auto mb-6">
+      The Hall of Fame members are the hardworking devs who have built
+      The Odin Project into the site that you see before you now.
+    </p>
 
-<div class="hall-of-fame gradient">
-  <h2 class="hall-of-fame__title text-2xl">Hall of Fame</h3>
```

Comment:
Didn't notice the first time through, but there was a sneaky <h2.5> in here :smile: